### PR TITLE
Revises telemetry figure to correct URL; reorders steps.

### LINKED ISSUE #1039: Revise telemetry URL, bare-metal-install-server, per same change in clr-installer.
Revise figure showing telemetry URL, bare-metal-install-server, per same change in clr-installer: https://github.com/clearlinux/clr-installer/pull/665

See Fig. 22 : https://docs.01.org/clearlinux/latest/get-started/bare-metal-install-server.html

### PR DESCRIPTION
**Pull request by mvincerx**
- Reorders Assign Hostname to match order of menus in UI
- Corrects typos and grammatically unclear sentences.
- Renames figures as a result of reordering steps, above.
- Replaces Fig 31, Swupd Mirror, to allow insecure connection.
- Closes #1039.

Signed-off-by: Michael Vincerra <michael.vincerra@intel.com>

### CODE REVIEW

File: source/get-started/bare-metal-install-server.rst [hunk 1]
```diff
@@ -810,36 +832,11 @@ If you have your own custom mirror of |CL|, you can add its URL.
...
       :scale: 100%
-      :alt: Assign Hostname
-
-      Figure 31: Assign Hostname
-
-#. Navigate to :kbd:`Confirm` until highlighted.
+      :alt: Swupd Mirror
 
-#. Select :kbd:`Confirm`.
+      Figure 31: Swupd Mirror
```
Comment (mdhorn) [inline source/get-started/bare-metal-install-server.rst]: Figure 31 needs to be recaptured to reflect that we now have a "Allow insecure http" checkbox item.

Comment (mvincerx) [reply source/get-started/bare-metal-install-server.rst]: Great catch. Will do.

Comment (mvincerx) [reply source/get-started/bare-metal-install-server.rst]: Figure 31 replaced now.  Does the new checkbox require explanation? IMHO, no, but please advise.

Comment (mdhorn) [reply source/get-started/bare-metal-install-server.rst]: No, I think it is self-explaining.

**Comment by mvincerx** [comment]
Thank you. LGTM.

**Pull request merged**